app/containers/TuneContainer/index.js

```diff
@@ -0,0 +1,128 @@
+/**
+ *
+ * TuneContainer
+ *
+ */
+
+import React, { memo, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import selectTuneContainer, { selectSearchTerm, selectSongsData, selectSongsError } from './selectors';
+import { tuneContainerCreators } from './reducer';
+import tuneContainerSaga from './saga';
+import { injectSaga } from 'redux-injectors';
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import { Input } from 'antd';
+import styled from 'styled-components';
+import debounce from 'lodash/debounce';
+
+const { Search } = Input;
+
+const Container = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    max-width: ${(props) => props.maxwidth}px;
+    width: 100%;
+    margin: 0 auto;
+    padding: ${(props) => props.padding}px;
+  }
+`;
+export function TuneContainer({
+  dispatchItuneSongs,
+  dispatchClearItuneSongs,
+  intl,
+  songsData = {},
```

Comment:
this was a problem with the react-template too, i made the changes. 

use defaultProps instead of adding default values here